### ISSUE TITLE
Add a new-theme command to setup the theme boilerplate

### ISSUE THREAD

**Issue by d0ugal** [mkdocs/mkdocs#652]
This will really help users after #530 lands.

**Comment by d0ugal** [comment]
I know we list the themes in the output of `mkdocs build --help` but it would be good to add it somewhere more obvious. So maybe we need a theme command group with sub commands like this?

    mkdocs theme new
    mkdocs theme list

**Comment by robodude666** [comment]
I do *like* the idea, however... I'm not certain this is a bug, nor that there needs to be a `new` command.

I see the value if it's intended to download, unzip, rename, etc. a `mkdocs-themeboilerplate` project to get less experienced users going... 

But if it'll only exist because we have `mkdocs theme [list]` and want more than one sub-command it's kind of unnecessary.

**Comment by d0ugal** [comment]
Good point. I have no idea why I marked this as a bug.

**Comment by pawamoy** [comment]
Nowadays I think it is better solved using project scaffolding/templating, for example with tools like [Copier](https://copier.readthedocs.io/en/stable/) and [Cookiecutter](https://pypi.org/project/cookiecutter/). Given the old age of this issue and the little interest it generated (few comments, no upvotes), we'll close this as out of scope for MkDocs core :slightly_smiling_face: Anyone is however and of course free to comment further or even pick up this task of creating a template project to kickstart new MkDocs themes :blush: I'll add an "Idea" label to issues like this one who could serve as ideas for third-party projects :slightly_smiling_face:
